courseplay.lua

```diff
@@ -293,23 +294,22 @@ local function setGlobalData()
 	[25]={name='self.cp.turnDiameter',dataFormat='Float'},
 	[26]={name='self.cp.coursePlayerNum',dataFormat='Int'}, --??
 	[27]={name='self.cp.laneOffset',dataFormat='Float'},
-	[28]={name='self.cp.toolOffsetX',dataFormat='Float'},
```

Comment:
@schwiti6190 same here, removed toolOffsetX from this list.

Reply:
The new setting gets synchronized automatically, so this one isn't needed anymore.